src/objects.rs

```diff
@@ -691,6 +691,44 @@ impl GetObjectType for VehicleJourney {
     }
 }
 
+#[derive(Debug)]
+pub enum StopTimeError {
+    DuplicateStopSequence { vj_id: String },
+    IncoherentStopTimes { vj_id: String },
+}
+
+impl std::fmt::Display for StopTimeError {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            StopTimeError::DuplicateStopSequence { vj_id } => {
+                write!(f, "duplicate stop_sequence for the trip '{}'", vj_id)
+            }
+            StopTimeError::IncoherentStopTimes { vj_id } => {
+                write!(f, "incoherent stop times for the trip '{}'", vj_id)
```

Comment:
The message doesn't make it clear that it is an ordering problem of the times. Here are a few suggestions:
- "stop times are not ordered correctly for the trip '{}'"
- "stop times are not strictly growing for the trip '{}'"

Reply:
These are the messages indicated in specifications